src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/BuildHostProcessManager.cs

```diff
@@ -35,10 +35,17 @@ public async Task<IBuildHost> GetBuildHostAsync(string projectFilePath, Cancella
 
         _logger?.LogTrace($"Choosing a build host of type {neededBuildHostKind} for {projectFilePath}.");
 
+        if (neededBuildHostKind == BuildHostProcessKind.Mono && MonoMSBuildDiscovery.GetMonoMSBuildDirectory() == null)
+        {
+            _logger?.LogWarning($"An installation of Mono could not be found; {projectFilePath} will be loaded with the .NET Core SDK and may encounter errors.");
```

Comment:
Not blocking this PR, but wonder if this would be worth a pop-up instead (with links to docs on how to acquire it?)  Same for .net framework case

Reply:
Same as other conversation: pop-up now added, and bug filed to track making docs.